.github/workflows/CI.yml

```diff
@@ -357,6 +355,52 @@ jobs:
       keptn-spec-version: ${{ needs.prepare_ci_run.outputs.KEPTN_SPEC_VERSION }}
       datetime: ${{ needs.prepare_ci_run.outputs.DATE }}${{ needs.prepare_ci_run.outputs.TIME }}
 
+  helm_charts_upload:
+    name: Publish helm charts to dev repo
+    needs: [prepare_ci_run, helm_charts_build]
+    if: (needs.prepare_ci_run.outputs.BUILD_EVERYTHING == 'true') || (needs.prepare_ci_run.outputs.BUILD_INSTALLER == 'true')
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout helm-charts repo
+      uses: actions/checkout@v2
+      with:
+        repository: keptn/helm-charts-dev
+        path: helm-charts-dev
+        ref: gh-pages
+        token: ${{ secrets.KEPTN_BOT_TOKEN }}
+
+    - name: Configure Git
+      working-directory: helm-charts-dev
+      env:
+        KEPTN_BOT_NAME: "Keptn Bot"
+        KEPTN_BOT_EMAIL: "keptn-bot <86361500+keptn-bot@users.noreply.github.com>"
+      run: |
+        git config user.name "$KEPTN_BOT_NAME"
+        git config user.email "$KEPTN_BOT_EMAIL"
+
+    - name: Install Helm
+      uses: azure/setup-helm@v1
+      with:
+        version: v3.7.2
```

Comment:
do we need a specific helm version? Can we make sure that the [build-helm-charts](https://github.com/keptn/keptn/blob/master/.github/workflows/build-helm-charts.yml) workflow uses the same helm version?

Reply:
not really. if it's left out, the latest stable version will be used

Reply:
actually, helm is included on ubuntu GH actions runner per default anyways so i will remove the setup step